apps/estuary/lib/estuary_web/views/error_helpers.ex

```diff
@@ -0,0 +1,65 @@
+defmodule EstuaryWeb.ErrorHelpers do
+  @moduledoc """
+  Conveniences for translating and building error messages.
+  """
+
+  use Phoenix.HTML
+
+  @doc """
+  Generates tag for inlined form input errors.
+  """
+  def error_tag(form, field) do
+    Enum.map(Keyword.get_values(form.errors, field), fn error ->
+      content_tag(:span, translate_error(error),
+        class: "error-msg",
+        id: "#{field}-error-msg",
+        data: [phx_error_for: input_id(form, field)]
+      )
+    end)
+  end
+
+  @doc """
+  Render an error_tag for the given input.
+
+  Fixes the bug with the date picker
+  not rendering the error message when clearing a valid
+  date using the date_input/3
+  https://elixirforum.com/t/liveview-phx-change-attribute-does-not-emit-event-on-input-text/21280
+  """
+  def error_tag_live(form, field) do
+    Enum.map(Keyword.get_values(form.errors, field), fn error ->
+      content_tag(:span, translate_error(error),
+        class: "error-msg",
+        id: "#{field}-error-msg"
+      )
+    end)
+  end
```

Comment:
You don't need this function.  We only had to add it to Andi once we started using the non text-input fields.  I don't think you need `error_tag` either unless you're doing form validation.